packages/component-life-span/index.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react"
+
+export const Counter = (() => {
```

Comment:
Used a closure here so I can reset the counter in the tests, otherwise I would have used a var.